nxcomp/src/Loop.cpp

```diff
@@ -3810,7 +3810,15 @@ int SetupAuthInstance()
 
         launchdAddrUnix.sun_family = AF_UNIX;
 
+        #ifdef __linux__
```

Comment:
We can hardcode this to 104 here without any additional checks or comments, since this code section will only be used on systems that have launchd (which really is just OS X and FreeBSD.)

I don't care though, can leave this as-is or amend.